CONTRIBUTING.md

```diff
@@ -0,0 +1,393 @@
+|-------------|---------|-----------------------------------------------------------------------------|
+| [Python](https://www.python.org/downloads/)      | `3.X`     | Required for running MkDocs and documentation-related tools.                |
+| [Node.js](https://nodejs.org/en/download/) / npm         | `Latest LTS` | Required for packaging and testing spectral rules.                         |
+| [Spectral](https://docs.stoplight.io/docs/spectral/b8391e051b7d8-installation)    | `Latest`  | Required for developing linting rules for OpenAPI specifications.          |
+
+You can install Python, npm and Spectral CLI using your system's package manager or download them from their respective websites.
+
+You can verify your installations with:
+
+```bash
+python --version
+node --version
+npm --version
+spectral --version
+```
+
+#### 4. Install MkDocs (required for documentation)
+
+You can view the guidelines directly in your markdown viewer of choices or use the same static site generator ([MkDocs](https://www.mkdocs.org/)) used to produce the github pages to serve the documentation locally.
+
+To install MkDocs your will require python 3.X once you have this you can install MkDocs and its plugins.
+
+MkDocs requires an `mkdocs.yml` file for configuration and navigation control, the one supplied in the repo is the one used for github pages but should work fine locally also.
+
+``` bash
+pip install mkdocs-material
+pip install markdown-callouts
+pip install mkdocs-git-revision-date-localized-plugin
+pip install mkdocs-git-committers-plugin-2
+pip install mkdocs-print-site-plugin
+pip install mkdocs-tech-docs-template
+pip install mkdocs-redirects
+pip install mkdocs-awesome-nav
+
+# Only needed if you want to generate site PDF locally
+pip install pytest-playwright
+playwright install --with-deps
+playwright install chrome --with-deps
+```
+
+### Understanding the Repository Structure
+
+- `/docs/` - Documentation content written in Markdown
+- `/docs/api-design-guidelines/` - API design guidelines content
+- `/docs/spectral-rules/` - Documentation for API linting rules
+- `/example/` - Example OpenAPI specifications
+- `/functions/` - JavaScript functions used in Spectral rules
+- `/overrides/` - MkDocs theme customisation files
+- `ukhsa.oas.rules.yml` - UKHSA-specific Spectral rules
+- `zalando.oas.rules.yml` - Zalando API guidelines rules
+
+## Contributing Process
+
+### Finding Issues to Work On
+
+- Check the [Issues](https://github.com/ukhsa-collaboration/api-guidelines/issues) section for open tasks
+- Look for issues tagged with `good first issue` if you're new to the project
+
+### Signed Commits
+
+All commits to this repository **MUST** be signed with a GPG key to verify the committer's identity. This helps ensure the security and integrity of the codebase.
+
+To set up signed commits:
+
+#### 1. Generate a GPG key (if you don't have one already)
```

Comment:
There are numberings that are a numbered list, and this one is a heading numbering. Please could you confirm that this is as intended?

Reply:
so they appear in the TOC at the top.